apps/neuvector/neuvector/aat/00.yaml

```diff
@@ -75,4 +73,8 @@ spec:
               Cluster_Name: aat00
       manager:
         ingress:
-          host: neuvector00.service.core-compute-aat.internal
+          host: cft-neuvector00.aat.platform.hmcts.net
+          enabled: true
```

Comment:
aat/00.yaml & aat/01.yaml are not similar? 

01.yaml for example has `path: "/"` etc